site/e2e/helpers.ts

```diff
@@ -149,25 +159,51 @@ export const verifyParameters = async (
   }
 };
 
+// StarterTemplates are ids of starter templates that can be used in place of
+// the responses payload. These starter templates will require real provisioners.
+export enum StarterTemplates {
+  STARTER_DOCKER = "docker",
+}
```

Comment:
We should probably add ways to add more to the templates later. For now, just using the starter template is easier.